### ISSUE TITLE
Feature Request: Enable timeshift to work with Fedora Btrfs implementation

### ISSUE THREAD

**Issue by BionicBlade0** [linuxmint/timeshift#148]
Fedora has a different btrfs implementation than Ubuntu. Fedora does not divide subvolumes with labels @ and @home. If timeshift can be compatible with their implementation, then more people would be using the app.

**Comment by zilexa** [comment]
+1

**Comment by geraldosimiao** [comment]
+1

**Comment by EmixamPP** [comment]
Unfortunately, timeshift is part of the Linux Mint project, I don't see why they would waste time making this kind of contribution. For Fedora, the only solution I can think of is snapper. Although, I would also strongly like timeshift to be compatible with the system used by Fedora. It's much more user-friendly to have a GUI.

**Comment by geraldosimiao** [comment]
Yeah, I know. I personally use snapper and btrfs-assistant as GUI. But, again, turning timeshift more "distro agnostic" would be very fine implementation.

**Comment by zilexa** [comment]
btrbk would be an even better solution. btrbk does not only do snapshots, also backups. It is very powerful and doesn't require snapshots to be created within the original subvolume folder (which is how Snapper works, that makes no sense to me..).

**Comment by sannidhyaroy** [comment]
+1

**Comment by tjkirch** [comment]
Not stale.
